resources/leiningen/new/xiana/src/backend/app_name.clj

```diff
@@ -1,41 +1,51 @@
 (ns {{sanitized-name}}
   (:require
-   [com.stuartsierra.component :as component]
-   [controllers.index :as index]
-   [controllers.re-frame :as re-frame]
-   [framework.components.app.core :as xiana.app]
-   [framework.components.router.core :as xiana.router]
-   [framework.components.web-server.core :as xiana.web-server]
-   [framework.config.core :as config]
-   [framework.db.storage :as db.storage]
-   [interceptors]
-   [reitit.ring :as ring]))
+    [com.stuartsierra.component :as component]
+    [controllers.index :as index]
+    [controllers.re-frame :as re-frame]
+    [framework.components.interceptors :as interceptors]
+    [framework.components.session.backend :as session-backend]
+    [framework.components.web-server.core :as xiana.web-server]
+    [framework.config.core :as config]
+    [framework.db.storage :as db.storage]
+    [nrepl.server :refer [start-server stop-server]]
```

Comment:
nrepl server shouldn't be here. Not used, and optional only.

Reply:
https://github.com/Flexiana/xiana-template/blob/ce4fecfe4d3d598c73e2dcb46353b8e1b1440345/resources/leiningen/new/xiana/src/backend/app_name.clj#L10-L11